package.json

```diff
@@ -664,8 +664,7 @@
           "ui-inventory.instance.view",
           "inventory.instances.item.post",
           "inventory-storage.instances.item.post",
-          "browse.contributors.instances.collection.get",
-          "consortia.inventory.local.sharing-instances.execute"
+          "browse.contributors.instances.collection.get"
```

Comment:
Is this a BE generated permission? I can't find it anywhere on the UI.
Also, there are 2 similar names: `consortia.inventory.local.sharing-instances.execute` which you've removed here. And there's `consortia.inventory.share.local.instance` that is used to define translations. Did we forget to update translations after changing permission names?

Reply:
Yes, `consortia.inventory.local.sharing-instances.execute` is generated on [BE side](https://github.com/folio-org/mod-consortia/blob/81eceb708352b5e5fd41a98d2ecdfbfaa77a96db/descriptors/ModuleDescriptor-template.json#L496)

Also updated the translation path for **Inventory: Share local instance with consortium** permission